tests/framework/Models/External/serialize_ext_model_and_use.xml

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" ?>
+<Simulation verbosity="debug">
+  <TestInfo>
+    <name>framework.Models.External.serialize_ext_model</name>
+    <author>alfoa</author>
+    <created>2020-09-26</created>
+    <classesTested>Models.ExternalModel</classesTested>
+    <description>
+      This test is used to exercise the ability or RAVEN to
+      serialize (pickle) the external model class and re-use it in
+      the framework.
+    </description>
+    <analytic>
+      This test uses the analytic model ''attenuate``, which is documented in the analytical test documentation.
+      Additionally, values of the ''from`` variables are exactly determined because they are set in the extmod methods:
+      \begin{itemize}
+        \item fromReadMoreXML: pi         (3.14159)
+        \item fromInit       : sqrt(pi)   (1.77245)
+        \item fromCNISelf    : pi/2       (1.57080)
+        \item fromCNIDict    : 2*sqrt(pi) (3.54491)
+      \end{itemize}
+      The exit strength ''ans`` is analytic with results as follows (Note that for testing purposes, the model always adds
+      0.05 to \texttt{y2} before calculating the exit strength):
+      \begin{itemize}
+        \item \texttt{y1}, \texttt{y2}, \texttt{ans}
+        \item 0.0, 0.0, 0.97531
+        \item 0.0, 1.0, 0.59156
+        \item 1.0, 0.0, 0.59156
+        \item 1.0, 1.0, 0.35880
+      \end{itemize}
+    </analytic>
+  </TestInfo>
+
+  <RunInfo>
+    <WorkingDir>SerializeWrkd</WorkingDir>
+    <Sequence>extSerialization,extSerielizedLoading,sample2,print</Sequence>
```

Comment:
Not a required change, but making the spelling of `Serialized` consistent would be a nice touch. :)